<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>Gene Delivery Using Lipid Nanoparticle-based Immunoengineering Approach | Kevin (Won June) Cho</title> <meta name="author" content="Kevin (Won June) Cho"> <meta name="description" content="Development of Targeted mRNA/pDNA Vaccines for Cancer/Malaria Prevention and Protection"> <meta name="keywords" content="deep-learning, image-segmentation, python, pytorch"> <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%E2%9A%9B%EF%B8%8F&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css"> <link rel="canonical" href="https://chokevin8.github.io/projects/3_project/"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/native.css" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js"></script> <script src="/assets/js/dark_mode.js"></script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/">Kevin (Won June) Cho</a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">About Me</a> </li> <li class="nav-item "> <a class="nav-link" href="/blog/">My Blog</a> </li> <li class="nav-item "> <a class="nav-link" href="/projects/">My Projects</a> </li> <li class="nav-item "> <a class="nav-link" href="/repositories/">My Github</a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">My Resume</a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fas fa-moon"></i> <i class="fas fa-sun"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="post-title">Gene Delivery Using Lipid Nanoparticle-based Immunoengineering Approach</h1> <p class="post-description">Development of Targeted mRNA/pDNA Vaccines for Cancer/Malaria Prevention and Protection</p> <p class="post-moredescription"><i>Undergraduate Research Assistant Project (2021 ~ 2022)</i></p> </header> <article> <hr> <h3 id="project-motivation--background"><strong>Project Motivation &amp; Background:</strong></h3> <p>Nucleic acid-based therapeutics have been emergent, as more and more nucleic acid therapeutics are being approved due to their ability to treat diseases by targeting the genetic blueprint itself. Four main types of nucleic acid-based therapeutics are: 1) antisense oligonucleotides (ASO), 2) ligand conjugated small interfering RNA (siRNA), 3) adeno-associated virus vectors (AAV), and 4) lipid nanoparticles (LNP). Recently, more and more nucleic acid-based therapeutics have been approved by the FDA, the most famous being the two COVID-19 vaccines during the pandemic (mRNA- based therapeutic). Recently, the FDA approved Adstiladrin, which is an AAV-based therapeutic for non-muscle-invasive bladder cancer (NMIBC), so more and more nucleic acid-based therapeutics are hitting the market in the oncology space as well.</p> <p>Compared to conventional therapeutics that usually target proteins that results in transient therapeutic effects, nucleic-acid based therapeutics are often much longer-lasting or even permanent depending on the nucleic acid used and the target. However, most nucleic-acid based therapies require a carrier, as they will be targeted by the immune system for degradation and clearance and therefore will cause unwanted inflammation and toxicity that can be detrimental or even fatal to the patient. <em>Therefore, this project’s research was focused on the design of the lipid nanoparticles, or LNPs, for effective delivery of pDNAs/mRNAs to develop vaccines for malaria/cancer.</em></p> <p>To briefly introduce the background of LNPs, LNPs usually are constituted of four components: 1) Ionizable lipid, 2) PEGylated lipid, 3) Cholesterol, and 4) Helper lipid. The first three components were fixed, and six different helper lipids were explored, with the ratios of the four lipids being varied as well. A total of 1080 different LNPs were designed, with two of the helper lipids being cationic, anionic, and zwitterionic. The ratio and the different helper lipids have a huge impact on the surface charge, pKa, and size of LNPs, which all have a huge influence in the successful delivery of the drug payload. <em>Therefore, the project’s main motivation is not only for successful delivery of the drug payload, but also about optimizing the formulation of the LNP beforehand.</em></p> <p><strong>To briefly introduce the two projects:</strong> <br> For the <strong><em>malaria pDNA vaccine project</em></strong>, the proposed route of administration was oral, or through the mouth. This makes sense, as it is a wide known fact that oral drugs experience a “first pass effect”, where the orally taken drugs go through the intestines and end up in the liver. This is why drugs that do not target cells in the liver cannot be taken orally as it is often times metabolized in the liver before it even has a chance to circulate in the bloodstream. In addition, the low pH environment of the stomach due to gastric acid also necessitates a delivery mechanism such as LNPs. However, because the target of the pDNA malaria vaccine is hepatocytes, an oral administration was fitting. Furthermore, pDNA is used rather than mRNA because pDNA has a more long-lasting effect than mRNAs do, however they are therefore more likely to elicit an unwanted immune response, which was known to be alleviated by co-delivery of anti-inflammatory siRNA.</p> <p>For the <strong><em>cancer mRNA vaccine project</em></strong>, the proposed method of administration was intramuscular, or through the muscle tissue. This is the same route of administration as the COVID-19 vaccines, as it is most effective to elicit a potent immune response. The cancer vaccine can either be prophylactic or therapeutic, but either way, it has to be able to deliver the mRNA that codes for the specific tumor-specific antigens (TSA) that the T cells can recognize and kill the tumor cells. In order to ensure this happens effectively, below is the proposed entire uptake and trafficking scheme of the mRNA LNPs:</p> <div class="row"> <div class="col-sm"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/3_project/mRNA_uptake-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/3_project/mRNA_uptake-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/3_project/mRNA_uptake-1400.webp"></source> <img src="/assets/img/3_project/mRNA_uptake.png" class="img-fluid rounded z-depth-1" width="auto" height="auto" title="mRNA uptake" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> </div> <div class="caption"> Proposed scheme showing the uptake and trafficking of mRNA LNPs. </div> <p>As we see above, mRNA LNPs are designed to enter the target antigen-presenting cells (APCs), which in our case are dendritic cells (DCs). They enter the cells via endocytosis, and it is important to design the LNPs in a way so they initiate <em>endosomal escape</em>, or else they are degraded by the lysosomes. After escaping the endosome by breaking the membrane of it, the mRNA is released in to the cytoplasm of the DCs, which initiates translation of the desired TSAs. Now, in DCs, DCs use the proteasome-TAP pathway as a main, conventional route for cross-presentation of the TSAs via MHC Class I molecules. In short, the translated TSA is broken into peptides by the proteasome, and the peptide is translocated by the TAP transporters into the lumen of the endoplasmic reticulum (ER). Then, the MHC Class I complex containing peptides and other molecules are assembled in the ER and transported to the cell surface. Lastly, as seen in the diagram, the CD8+ T-cells recognize the MHC Class 1 complex/molecule via its surface receptor, or T cell receptor (TCR). When this cross-presentation is successful, with the help of other signals called co-stimulary signals, the CD8+ T cell is <em>activated</em>, meaning it can now detect and kill target tumor cells via recognizing their TSAs.</p> <hr> <h3 id="methods"><strong>Methods:</strong></h3> <p>The methods for screening the 1080 LNPs for the pDNA malaria vaccine project are shown in below diagram.</p> <div class="row"> <div class="col-sm"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/3_project/lnp_optimization_method.PNG-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/3_project/lnp_optimization_method.PNG-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/3_project/lnp_optimization_method.PNG-1400.webp"></source> <img src="/assets/img/3_project/lnp_optimization_method.PNG" class="img-fluid rounded z-depth-1" width="auto" height="auto" title="LNP optimization" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> </div> <div class="caption"> Proposed method to screen 1080 LNPs of different lipid composition and helper lipid types. </div> <p><strong><em>Screening Process:</em></strong> <br> <strong>1.</strong> Component and ratio screening was done via <em>in vitro</em> transfection for each of the 1080 LNPs. Each LNPs were synthesized and tested for pDNA transfection in human HepG2 cells via using 50% mCherry and 50% Luc pDNA. Then, subsequent luciferase protein expression (more pDNA delivered, more bioluminescence in the cells) was tested and as a result, top 32 formulations for each of the six helper lipids were selected. <br> <strong>2.</strong> An <em>in vivo</em> screening via intrahepatic injection was performed, but in <em>batch mode</em>. Just like in COVID-19 testing, where multiple patients’ samples are merged together to test for COVID, the same logic was applied here. 8 of the top 32 formulations were grouped, meaning there were four clusters per helper lipid with a total of 24 clusters tested for intrahepatic injection. The same mix of mCherry and Luc pDNA were injected, and transfection efficiency were determined by imaging the mice liver via in vivo imaging system (IVIS). Furthermore, flow cytometry was used to measure mCherry levels in different types of cells in the liver to see which cells were transfected. As a result, top 12 clusters were selected for having the highest bioluminescence in the liver. <br> <strong>3.</strong> Again, subsequent <em>in vivo</em> screenings via intravenous injection was performed, which was in batch mode as well. The same procedure as before was repeated twice, until only top four formulations remained. <br> <strong><em>Post-Screening Process:</em></strong> <br> Then, after the screening process was over that left us with top four formulations, a final <em>in vivo</em> screening was performed via using Cre-Ai9 mouse. A Cre-Ai9 mouse, when injected with a Cre pDNA loaded LNP, allows the expression of tdTom only if the pDNA is successfully delivered. Following experiments using confocal imaging and flow cytometry was used to compare the top four formulations effectiveness. Lastly,</p> <p>The above method was for the pDNA malaria vaccine project, and more details and further experiments are explained in the published paper <a href="https://www.nature.com/articles/s41467-022-31993-y" rel="external nofollow noopener" target="_blank">here</a>. For the mRNA cancer vaccine project, the same screening process can be used, but just in a different applied method: <br> <strong><em>Screening Process:</em></strong> <br> <strong>1.</strong> Component and ratio screening was done via <em>in vitro</em> transfection for the same 1080 LNPs. Each LNPs were synthesized, and tested for mRNA transfection in bone-marrow dendritic cells (BMDCs) instead, since the target is DCs. Then, after waiting for a few days, we collect the BMDCs and co-culture them with CD8+ T cell. Then, we measure level of cell proliferation to test if MHC class I cross-presentation between BMDCs and CD8+ T cells happened or not. <br> <strong>2.</strong> An <em>in vivo</em> screening via subcutaneous/intramuscular injection was performed, also in batch mode. Screening based on measuring cell proliferation- cell proliferation will be evident if T-cells are activated by BMDCs. <br> <strong>3.</strong> Again, subsequent <em>in vivo</em> screenings in batch mode until top four formulations remain. <br> <br> <strong><em>Post-Screening Process:</em></strong> <br> Then, after the screening process was over that left us with top four formulations, a final <em>in vivo</em> screening was performed via using the Cre-Ai9 mouse as well.</p> <p><strong><em>Below is a summary of the main work and contributions that I did to each of the two projects, with a list of other detailed experiments not mentioned for brevity.</em></strong> <br></p> <ol> <li> <strong>In pDNA malaria vaccine project:</strong> <ul> <li>Formulated and screened ~1080 LNPs via cluster-mode in vitro transfection assays and in vivo intrahepatic/intraduodenal injections.</li> <li>Tested Ai9 Cre reporter mice for co-delivery of anti-inflammatory siRNA with pDNA for in vivo assays of top formulations. <br> List of other experiments not explicitly mentioned above:</li> <li> <em>In vivo</em> experiment with using OVA-SIINFEKL model antigen to determine immunological effects.</li> <li> <em>In vivo</em> experiment using Cy5 stain to look at LNP biodistribution.</li> <li>ELISA for measuring STAT and NF-κB levels within the liver to determine inflammation levels. <br> </li> </ul> </li> <li> <strong>In mRNA cancer vaccine project:</strong> <ul> <li>Formulated and screened ~1080 LNPs via cluster-mode in vitro transfection assays on dendritic cells, and in vivo subcutaneous/intramuscular injections.</li> <li>Tested Ai9 Cre reporter mice for mRNA delivery and participated in subsequent therapeutic and prophylactic mice tumor studies. <br> </li> </ul> </li> </ol> <p>List of other experiments not explicitly mentioned above:</p> <ul> <li> <em>In vivo</em> experiment with using OVA-SIINFEKL model antigen to determine immunological effects.</li> <li> <em>In vivo</em> experiment using Cy5 stain to look at LNP biodistribution.</li> <li>CCK-8 for measure of proliferation in co-culture of BMDCs and CD8+ T cells.</li> <li>ELISA for measuring overall cytokine concentration in the following cell culture supernatant and ELISPOT for frequency of cytokine release.</li> <li>ICS (intracellular cytokine staining) followed by flow cytometry measuring cytokine concentration at a cell-type specific manner.</li> </ul> <p><strong><em>Note that I participated in group discussions/experiments of designing/performing all of the experiments shown above.</em></strong></p> <hr> <h3 id="results"><strong>Results:</strong></h3> <p>The results of the <a href="https://www.nature.com/articles/s41467-022-31993-y" rel="external nofollow noopener" target="_blank">published paper</a> were mainly the fact that the batch-mode screening process is effective in selecting top performing formulations out of a huge library of LNPs. Furthermore, another major result is that co-delivery of anti-inflammatory siRNA with pDNA alleviates the unwanted inflammation caused by the pDNA and the adjuvant effect of LNPs and therefore extends the duration of transgene expression via pDNA. This is possible due to the STAT and NF-κB siRNA being co-delivered with the pDNA as a payload.</p> <p><em>Note that I cannot reveal the detailed methods and results for the mRNA cancer vaccine project since it has not been published yet.</em></p> <hr> <h3 id="personal-comments"><strong>Personal Comments:</strong></h3> <h3 id="q-why-did-i-choose-this-project">Q: Why did I choose this project?</h3> <p> Due to the COVID-19 outbreak, my start to undergraduate research was late, but the mRNA COVID-19 vaccine sparked my interests in gene delivery, particularly for therapeutic gene delivery mechanisms for a prophylactic/therapeutic cancer vaccine. Therefore, participating in these two projects was perfectly in-line with my interests at that time. Furthermore, utilizing and activating the human immune system for preventing/reversing cancer just sounded really cool, as other similar therapies such as CAR-T or immune checkpoint inhibitors were the new "hot" immunotherapies in the oncology space around that time. </p> <h3 id="q-what-did-i-do-outside-of-this-project">Q: What did I do outside of this project?</h3> <p> Before joining this lab, I did not have any practical experience in the wet lab, while I did have some background knowledge regarding LNPs because I took relevant courses in school (ex. drug delivery, supramolecular materials, etc). Therefore, I made sure to review the experimental protocols and analyzed why an experiment was planned the way it is. Lastly, reading articles (journal club) regarding LNPs also helped me to stay up to date with recent discoveries/techniques. </p> <h3 id="q-what-impact-did-this-project-have-on-me">Q: What impact did this project have on me?</h3> <p> Thanks to the kind guidance of the group in the lab, I had the privilege of designing and performing various different wet lab techniques as summarized above. Even though I'm currently doing computational work, I believe the so-called "scientific method" is equal whether it's in the wet lab or the dry lab. Therefore, this has really taught me how to effectively set up an experiment and execute it in the most efficient way. Also, I believe the relative oncology and immunobiology/engineering background that I gained with this experience will eventually help me in the future, even though I've switched over to dry lab. Latly, while not one of the principal authors of the paper, this was my first time contributing to performing experiments and writing a manuscript to be submitted for publication. </p> <hr> <p><em>Image credits to:</em></p> <ul> <li>Scheme of uptake and trafficking of mRNA LNPs (In-house)</li> <li><a href="https://www.nature.com/articles/s41467-022-31993-y" rel="external nofollow noopener" target="_blank">Method to screen 1080 LNPs</a></li> </ul> </article> </div> </div> <footer class="fixed-bottom"> <div class="container mt-0"> © Copyright 2023 Kevin (Won June) Cho. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.bundle.min.js" integrity="sha256-fgLAgv7fyCGopR/gBNq2iW3ZKIdqIcyshnUULC4vex8=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js"></script> <script defer src="/assets/js/common.js"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>